util/dvsim.py

```diff
@@ -437,6 +437,8 @@ def main():
     # and other ASIC flow targets.
     if args.tool == 'ascentlint':
         cfg = LintCfg.LintCfg(args.cfg, get_proj_root(), args)
+    elif args.tool == 'dc':
+        cfg = SynCfg.LintCfg(args.cfg, get_proj_root(), args)
```

Comment:
SynCfg.LintCfg looks weird to me, and we have SimCfg.SimCfg below ...

Reply:
Good point. This is actually wrong and should not have made it into this PR. It is corrected in #1749, which performs the dvsim integration. Let me remove this particular change.